### ISSUE TITLE
Release v4.4.2 video stream is black (doesn't show) in detached window

### ISSUE THREAD

**Issue by kwh44** [mavlink/qgroundcontrol#11857]
----

## Expected Behavior
I launched QGC, video stream is shown in small embedded window, but when clicking on detaching video stream into separate window, video stream is expected to show in detached window as in embedded (small resizable screen on top of map)

## Current Behavior
Please describe what happens instead of the expected behavior.
I launched QGC, video stream is shown in small embedded window, but when clicking on detaching video stream into separate window, video stream is black, doesn't show in that window

## Steps to Reproduce:
Please provide an unambiguous set of steps to reproduce the current behavior
1. Launch QGC
2. Wait for video stream to show in small embedded window
3. Click on button to detach video stream into separate window
4. Now you can see that video stream is black in detached window, while in small embedded window it was showing the stream.

## System Information
When posting bug reports, include the following information
- Operating System: Ubuntu 24.04.1 LTS
- QGC Version: 4.4.2 official build
- QGC build: downloaded from https://github.com/mavlink/qgroundcontrol/releases/download/v4.4.2/QGroundControl.AppImage
- Flight Controller: SITL
- Autopilot (with version):  SITL ardupilot

## Detailed Description
video stream doesn't show in detached window, I am using QT 5.15.2, gstreamer version is  1.24.2

## Log Files and Screenshots
Video stream is shown in embedded window
![Screenshot from 2024-09-08 15-38-51](https://github.com/user-attachments/assets/e2b6640d-c337-4f6b-a277-213452638b38)
Video stream is not shown (black) in detached window
![Screenshot from 2024-09-08 15-39-24](https://github.com/user-attachments/assets/c7e00f5d-5fda-4788-8b1b-2c9a4c692180)

**Comment by kwh44** [comment]
https://github.com/mavlink/qgroundcontrol/issues/10803 looks to be previously reported

**Comment by aXynt** [comment]
I also have the same problem on ubuntu 22.04 and 24.04 with v4.4 or daily version. @HTRamsey do you have any solution?

**Comment by HTRamsey** [comment]
Yeah it was one of the things I was going to fix before v5. Probably will focus on mostly video in general for a while

**Comment by aXynt** [comment]
@HTRamsey 5.0 stable is coming soon, will you fix this for 5.0?

**Comment by HTRamsey** [comment]
Somehow this is fixed now
